django_ilmoitin/api/schema/types.py

```diff
@@ -0,0 +1,45 @@
+import graphene
+from django.apps import apps
+from django.conf import settings
+from graphene_django import DjangoObjectType
+
+from django_ilmoitin.models import NotificationTemplate
+from django_ilmoitin.utils import render_preview
+
+LanguageEnum = graphene.Enum(
+    "Language", [(lang[0].upper(), lang[0]) for lang in settings.LANGUAGES]
+)
+
+
+class NotificationTranslationType(DjangoObjectType):
+    language_code = LanguageEnum(required=True)
+    subject = graphene.String()
+    body_html = graphene.String()
+    body_text = graphene.String()
+    preview = graphene.String()
```

Comment:
it doesn't need a resolver on this type? 🤔

Reply:
WHOOPS that actually shouldn't be there. 😅 

Or should it... 🤔 Would it make sense to have a preview for every possible language instead of a "global" notification-level preview?

Reply:
I think it would be beneficial to have a preview for every language. and then global-level preview would be for the default language 🤔